app/views/listings/_google_places_form.html.haml

```diff
@@ -30,7 +30,7 @@
         %hr
         .input-group.mb-5
           = label_tag "Add Categories"
-          = select_tag "categories[]", options_for_select(@categories),{ class: "ui fluid search dropdown", id: "categories", :multiple => "", :prompt => 'Search' }
+          = select_tag "categories[]", options_for_select(@all_categories,  @listing_categories),{ class: "ui fluid search dropdown", id: "categories", :multiple => "", :prompt => 'Search' }
```

Comment:
I might have gone for `options_for_select(@categories, @listing.categories)` here. Also, might be good to get in the practice of writing a test to verify failure when we fix a bug.

Reply:
Ah, I see. I wonder if we could use `collection_select` here instead?

http://api.rubyonrails.org/classes/ActionView/Helpers/FormOptionsHelper.html